website/pages/docs/drivers/external/podman.mdx

```diff
@@ -175,6 +181,15 @@ config {
 }
 ```
 
+- `network_mode` - Set the [network mode][network-mode] for the container.
```

Comment:
Hm, will this override the group network? If so we should probably clarify that.

`container:id` seems difficult if not impossible to use manually and may only be useful for Nomad to use internally for group networking.

Reply:
I can remove it from the list, will add a note, network_mode will get overridden by a passed in nomad namespace (for connect)